src/ARTED/modules/opt_variables.f90

```diff
@@ -204,6 +212,27 @@ subroutine init_for_padding
     end do
   end subroutine
 
+  subroutine init_projector(zJxyz)
+    use global_variables
+    implicit none
+    integer,intent(in) :: zJxyz(Nps,NI)
+    integer :: i,ioffset
+
+    NP = Nlma / NI
```

Comment:
The number of projectors may be different for different ions. Thus, `NP` can be different from `Nlma/NI`.